app/views/course/leaderboards/groups.html.slim

```diff
@@ -0,0 +1,13 @@
+= page_header format_inline_text(@leaderboard_settings.group_leaderboard_title || t('.header'))
+
+= render partial: 'tabs'
+
+- course_groups_points = @course_groups.ordered_by_experience_points.take(display_user_count)
+= render layout: 'table', locals: { header: t('.experience_points_header'),
```

Comment:
@weiqingtoh Sorry, I'd rather not defining a partial for table, this makes the view not clear and we didn't benefit much from it ( you need to pass header class to it), how do you think?